FrontEnd/src/components/AboutPage/MeetTheTeam/MemberInfoCard/MemberInfoCard.stories.tsx

```diff
@@ -11,4 +12,5 @@ const Template: Story<ComponentProps<typeof MemberInfoCard>> = (args) => (
   <MemberInfoCard {...args} />
 );
 
-// export const Default = Template.bind({});
+export const CardList = MeetTheTeam;
```

Comment:
The purpose of this file is to hold the story of MemberInfoCard. If you want a story for MeetTheTeam, you should make a new file for it